js/components/operatorSignIn/list.tsx

```diff
@@ -0,0 +1,62 @@
+import { back } from "../../browser";
+import { findEmployeeByBadge, useEmployees } from "../../hooks/useEmployees";
+import { useSignins } from "../../hooks/useSignIns";
+import { fullName } from "../../models/employee";
+import { DateTime } from "luxon";
+import { ReactElement } from "react";
+
+export const List = ({ line }: { line: string }): ReactElement => {
```

Comment:
*suggestion:** this could be a `HeavyRailLine` type or something like that, rather than an arbitrary `string`.

Reply:
ec7c168